tox.ini

```diff
@@ -74,7 +74,8 @@ commands =
     {toxinidir}/scripts/run_regression.sh
 deps =
     unittest2
-#   Use a development checkout of oauth2client until a release is made
-#   which fixes https://github.com/google/oauth2client/issues/125
-    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
At least in the past, tox has been known to install the package-under-test without all its declared dependencies, unless they are declared explicitly in `tox.ini`.  If `tox --recreate` doesn't fail with your change, then I guess we're OK here.